addons/account/models/account.py

```diff
@@ -705,7 +705,8 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
                 taxes += ret['taxes']
                 continue
 
-            tax_amount = tax._compute_amount(base, price_unit, quantity, product, partner)
+            tax_excluded = tax_included = tax_base = base
```

Comment:
It is needed to have it on an isolated level which you can see if you take a look at the key of account.invoice.tax which can split taxes of the same type in two or even more entries and might be seeded by several invoice lines.